tests/specs/cli/commits.ts

```diff
@@ -226,4 +184,30 @@ export default testSuite(({ describe }) => {
 			});
 		});
 	});
+
+	function selectYesOptionAICommit(committing: ExecaChildProcess<string>) {
+		committing.stdout!.on('data', (buffer: Buffer) => {
+			const stdout = buffer.toString();
+			if (stdout.match('└')) {
+				committing.stdin?.write('y');
+				committing.stdin?.end();
+			}
+		});
+	}
+
+	function hitEnterToAcceptCommitMessage(committing: ExecaChildProcess<string>) {
+		committing.stdout!.on('data', function onPrompt(buffer: Buffer) {
+			const stdout = buffer.toString();
+			if (stdout.match('└')) {
+				committing.stdin!.write('\r');
+				committing.stdin!.end();
+				committing.stdout?.off('data', onPrompt);
+			}
+		});
+	}
+
+	async function getGitStatus(git: GitType): Promise<string> {
+		const statusBefore = await git('status', ['--porcelain', '--untracked-files=no']);
+		return statusBefore.stdout;
+	}
```

Comment:
I actually prefer the verbosity of being able to see the exact flags that are passed in, and the flexibility of being able to make one-off changes where appropriate.